sharding/collation.go

```diff
@@ -104,13 +104,43 @@ func (c *Collation) ProposerAddress() *common.Address {
 
 // CalculateChunkRoot updates the collation header's chunk root based on the body.
 func (c *Collation) CalculateChunkRoot() {
-	// TODO: this needs to be based on blob serialization.
-	// For proof of custody we need to split chunks (body) into chunk + salt and
-	// take the merkle root of that.
-	chunkRoot := common.BytesToHash(c.body)
+	chunks := Chunks(c.body)             // wrapper allowing us to merklizing the chunks.
```

Comment:
Is there a more idiomatic way this can be done? sounds a bit redundant. Perhaps BytesToChunks?